openquake/calculators/event_based_risk.py

```diff
@@ -200,13 +209,12 @@ class EventBasedRiskCalculator(base.RiskCalculator):
     core_func = event_based_risk
 
     epsilon_matrix = datastore.persistent_attribute('epsilon_matrix')
-    spec_indices = datastore.persistent_attribute('spec_indices')
     is_stochastic = True
 
     outs = collections.OrderedDict(
         [('AVGLOSS', 'avg_losses-rlzs'),
-         ('AGGLOSS', 'agg_losses-rlzs'),
-         ('SPECLOSS', 'specific-losses-rlzs'),
+         ('AGGLOSS', 'agg_losses'),
```

Comment:
It might be better to call this `agg_loss_table` so as to use the same terminology as the `asset_loss_table`